migrating_from_ocp_3_to_4/premigration-checklists-3-4.adoc

```diff
@@ -41,13 +50,7 @@ $ oc get pods --all-namespaces --field-selector=status.phase=Running \
 +
 Even if the pods are in a *Running* state, a high restart count might indicate underlying problems.
 
-* [ ] You have deleted old images by running the following command:
-+
-[source,terminal]
-----
-$ oc adm prune images
-----
-
+* [ ] You have removed old builds, deployments, and images from each namespace to be migrated by xref:../applications/pruning-objects.adoc#pruning-objects[pruning].
```

Comment:
Why would you use an xref instead of a command? I think it is less user-friendly.

Reply:
Because the actual command includes <options>, and there are lots of explanations of what they do.